neural_compressor/model/onnx_model.py

```diff
@@ -66,6 +61,23 @@ def __init__(self, model, **kwargs):
         self._get_graph_info()
         self._q_config = None
 
+    def check_large_model(self):
+        """Check model > 2GB."""
+        init_size = 0
+        for init in self._model.graph.initializer:
+            # if size of the initializer > 2GB, return True
+            try:
+                init_bytes = init.SerializeToString()
```

Comment:
if the initializer is saved to external file, can you still get its size with this method?

Reply:
Yes. `self._model = model if not isinstance(model, str) else onnx.load(model)` will load initializer data

Reply:
but if users pass loaded model object to this class, it will not execute onnx.load(model)

Reply:
update check of initializer data_location